umm-spec-lib/src/cmr/umm_spec/json_schema.clj

```diff
@@ -12,7 +12,8 @@
    [cmr.umm-spec.versioning :as ver]))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;; Defined schema files
+;;; Defined schema files
```

Comment:
When should we use 3 semi-colons?

Our coding style right now has:
https://wiki.earthdata.nasa.gov/display/CMR/Coding+Style+Guidelines

Use two semicolons for a comment on it's own line.
Use one semicolon for a comment at the end of a line.

Most of the places I see changed here seem like they should be 2 to stay consistent.

Reply:
* 4 semi-colons is for the top of a file for whole module/ns notes
* 3 is for module/ns-level notes pertinent to a section
* 2 is for function-level notes
* 1 is for end of line notes

Common Lisp set this pattern and when Google bought out one of the biggest Lisp companies (air line reservation software) their new Lispers started publishing some nice Lisp docs. Semi-colons are addressed here:
 * https://google.github.io/styleguide/lispguide.xml#Comment_semicolons

The Clojure style guide borrows from this long-standing Lisp tradition:
 * https://github.com/bbatsov/clojure-style-guide

Reply:
Sounds good to me, can you update our wiki page?

Reply:
Done.